src/Humanizer/Localisation/TimeToClockNotation/DeTimeOnlyToClockNotationConverter.cs

```diff
@@ -0,0 +1,93 @@
+﻿#if NET6_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Humanizer.Localisation.TimeToClockNotation;
+
+public class DeTimeOnlyToClockNotationConverter : ITimeOnlyToClockNotationConverter
+{
+    /// <summary> Switch to output Digits as Words </summary>
+    public bool AsWords;
+
+    /// <summary> Used to pre-pend, append or omit the Quarter of the Day </summary>
+    public bool? PrePendQuarter = false;
+
+    public string Convert(TimeOnly time, ClockNotationRounding roundToNearestFive)
+    {
+        var ret = time.AsClockDe(roundToNearestFive, AsWords);
+        if (!PrePendQuarter.HasValue
+            || ret == German.MidNight
+            || ret == German.MidDay)
+            return ret;
+
+        return PrePendQuarter.Value switch
+        {
+            false => ret + ' ' + time.GetDayQuarterGermanGenitive(),
+            true => time.GetDayQuarterGermanGenitive() + ' ' + ret
+        };
+    }
+}
+
+public static class German
```

Comment:
Can we nest this in `DeTimeOnlyToClockNotationConverter`? I think it shouldn't be part of the public API, either.

Reply:
You could use these static Methods when you know you're dealing with German. 
That would save you of handling Registration. 
I would even like move this to its own File so you can include it from other projects when you know it's for de-*
What do you think of that?